docs/json/radarr/cf/dv-hdr10.json

```diff
@@ -6,12 +6,21 @@
   "includeCustomFormatWhenRenaming": false,
   "specifications": [
     {
-      "name": "DV HDR10",
+      "name": "DV",
       "implementation": "ReleaseTitleSpecification",
       "negate": false,
       "required": true,
       "fields": {
-        "value": "^(?=.*\\b(DV|DoVi|Dolby[ .]?Vision)\\b)(?=.*\\b(HDR(10)?(?!\\+))\\b)"
```

Comment:
Rather than a double lookahead, match two specifications, DV and HDR10

Reply:
There is a reason why we have them separate
1. The releases often use DV even that the release has DV HDR10
2. When Radarr probes it it will use DV HDR10, so with the change you've done (just did a quick check on my phone) it could result in a download loop

Reply:
I'm not 100% sure I follow. The case you're mentioning is
> The release title specifies `DV` but the content is actually `DV HDR10`; radarr probles and puts `DV HDR10` in the filename (on rename). At the end of the operation the file is flagged as `DV HDR10` but shouldn't?

The other part I'm not sure I understand is how materially different is the lookup of both `DV` and `HDR10` in a single specification versus in two specifications.
Both of them are required; the behaviour should be identical.

Reply:
the line comparison on github looked a but weird on my phone, and it looked like if you renamed line 5 from the screenshot
![image](https://user-images.githubusercontent.com/6155095/226163004-301a75c3-6ca9-4b44-a079-d5fbce432e7d.png)

Reply:
> Rather than a double lookahead, match two specifications, DV and HDR10

there are several ways to approach the same result, you can add them all separate you can combine them as 1 liner, you can even setup the regex differently to gain the same results.
i'm sure we had them all as separate conditions, but in the past time we where trying to work around a issue what i personally think is a Radarr/Sonarr v4 glitch, and we won't be able to fix our self.

We're always open for improvement and any help possible, normally we discuss issues in our Private Discord dev support channel. because it's much faster to interact then on github.

Reply:
Happy to be added in if it helps with the conversation!